src/main/java/tweeting/services/Twitter4JService.java

```diff
@@ -99,10 +142,11 @@ private Tweet constructTweet(Status status) {
         return tweet;
     }
 
-    private List<Tweet> constructTweetList(List<Status> statuses) {
-        return statuses.stream()
-                .map(this::constructTweet)
+    public static List<Tweet> constructTweetList(List<Status> statuses) {
+        return statuses
```

Comment:
this looks like it is only constructing tweets and doing nothing else, use the lambda syntax above vs sub function